easybuild/easyconfigs/s/shapeit/shapeit-2.r837.GLIBCv2.12_linux_x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
+versionsuffix = '_linux_x86_64'
```

Comment:
@rtripath89 is this something so slapped on yourself, or do you have a reference for it?

It doesn't seem to be adding much, so maybe this should be dropped?